src/array/cuda/spmm.cu

```diff
@@ -612,17 +612,17 @@ void SpMMCsrHetero(const std::string& op, const std::string& reduce,
                 NullArray() : vec_ufeat[src_id];
             NDArray efeat = (vec_efeat.size() == 0) ?
                 NullArray() : vec_efeat[etype];
-            cuda::SpMMCsr<IdType, DType, Op, cuda::reduce::Max<IdType, DType> >(
-                bcast, csr, ufeat, efeat, vec_out[dst_id], out_aux[0], out_aux[1]);
+            // cuda::SpMMCsr<IdType, DType, Op, cuda::reduce::Max<IdType, DType> >(
+            //     bcast, csr, ufeat, efeat, vec_out[dst_id], out_aux[0], out_aux[1]);
```

Comment:
I think it's better to remove these two blocks if they are not implemented.

Reply:
Removed for now.